M2/Macaulay2/packages/Varieties.m2

```diff
@@ -175,6 +228,14 @@ describe ProjectiveVariety := X -> Describe (expression Proj) (expression X.ring
 -- Divisors
 -----------------------------------------------------------------------------
 
+-- overriding the dummy methods defined in Truncations package
+nefCone       Ring := R -> if R.?variety then nefCone       R.variety
+nefGenerators Ring := R -> if R.?variety then nefGenerators R.variety
+
```

Comment:
Since a ring doesn't generally have a well-defined notion of a nef cone, this kludge is unfortunate.  Can we not think of a better way to handle this issue?